nbs/12_gsa_gls/py/99-gls-export.py

```diff
@@ -0,0 +1,144 @@
+# ---
+# jupyter:
+#   jupytext:
+#     cell_metadata_filter: all,-execution,-papermill,-trusted
+#     formats: ipynb,py//py:percent
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.7.1
+#   kernelspec:
+#     display_name: Python 3
+#     language: python
+#     name: python3
+# ---
+
+# %% [markdown] tags=[]
+# # Description
+
+# %% [markdown] tags=[]
+# This notebook exports results into other more accessible data formats.
+
+# %% [markdown] tags=[]
+# # Modules
+
+# %% tags=[]
+import pandas as pd
+import rpy2.robjects as ro
+from rpy2.robjects import pandas2ri
+from rpy2.robjects.conversion import localconverter
+
+import conf
+
+# %%
+readRDS = ro.r["readRDS"]
+
+# %% tags=[]
+saveRDS = ro.r["saveRDS"]
+
+# %% [markdown] tags=[]
+# # Settings
+
+# %% tags=[]
+OUTPUT_DIR = conf.RESULTS["GLS"]
+display(OUTPUT_DIR)
+
+assert OUTPUT_DIR.exists()
+
+# %% [markdown] tags=[]
+# # Get results files
+
+# %%
+input_filepath = OUTPUT_DIR / "gls_phenotypes-combined-phenomexcan.pkl"
+display(input_filepath)
+
+# %%
+data = pd.read_pickle(input_filepath)
+
+# %% tags=[]
+data.shape
+
+# %% tags=[]
+data.head()
+
+# %%
+assert not data.isna().any().any()
+
+# %% [markdown] tags=[]
+# # Save
+
+# %% [markdown] tags=[]
+# ## RDS format
+
+# %% tags=[]
+output_file = input_filepath.with_suffix(".rds")
+display(output_file)
+
+# %%
+with localconverter(ro.default_converter + pandas2ri.converter):
```

Comment:
What does localconverter do? Is it like a scope specific python -> R dataframe converter?

Reply:
Yes, it comes from `rpy2`, and it's a context manager where you convert from python to R or vice-versa.

Thank you, @danich1!